src/test/kotlin/StartupTimeTest.kt

```diff
@@ -220,7 +236,8 @@ class StartupTimeTest : TestBase() {
                 }
 
                 val logEntries = driver.manage().logs().get("logcat")
-                val regex = Regex("Displayed ${app.name}/\\.${app.activity}: \\+(?:([0-9]+)s)?([0-9]+)ms")
+                val trackedActivity = app.measureActivity ?: app.activity
+                val regex = Regex("Displayed ${app.name}/\\.$trackedActivity: \\+(?:([0-9]+)s)?([0-9]+)ms")
```

Comment:
Note that the time (ms) grabbed from the `measureActivity` is probably just the activity launch time and not the app launch time. The later needs to be calculated or grabbed in another way.